firmware/main.c

```diff
@@ -104,6 +104,9 @@ int main(void) {
     port_init(&port_b, 2, &PORT_B, GCLK_PORT_B,
         TCC_PORT_B, DMA_PORT_B_TX, DMA_PORT_B_RX);
 
+    // TEMPORARY
+    ws2812_init(PORT_B.power);
```

Comment:
@rwaldron I was just using this to check that my makefile changes worked and I could call the functions I wrote :) It won't be on by default in production.